pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[tool.black]
+target-version = ["py39"]
+
+[tool.ruff]
+select = ["E", "F", "C4", "I001", "UP", "SIM"]
```

Comment:
This enables the closest set of rules that `shed` fixed without being overly specific.

- `F`: pyflakes/autoflake
- [`C4`: flake8-comprehensions](https://beta.ruff.rs/docs/rules/#flake8-comprehensions-c4) - shed has some custom autofixers
- `I001`: isort
- `UP`: pyupgrade
- [`SIM`: flake8-simplify](https://beta.ruff.rs/docs/rules/#flake8-simplify-sim) - I think shed has a couple of these autofixers